camunda-formio-tasklist-vue/CHANGELOG.md

```diff
@@ -0,0 +1,24 @@
+# Changes for camunda-formio-tasklist-vue
+
+## v1.0.6
```

Comment:
Need to change this to v1.0.7 later - 30-06-2021

Reply:
Also the package.json and package-lock.json were not updated with latest version